obal/data/roles/diff_package/tasks/koji.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- set_fact:
+    koji_tags: "{{ diff_package_tags }}"
+  when: diff_package_tags is defined
```

Comment:
Keeps backwards compatibility. I debated whether we should lock our older branches to a version of Obal instead and keep the code clean going forward.